metacat-main/src/main/java/com/netflix/metacat/main/services/impl/TableServiceImpl.java

```diff
@@ -509,6 +512,26 @@ public TableDto updateAndReturn(final QualifiedName name, final TableDto tableDt
         }
     }
 
+    /**
+     * Logs the get-table parameters as spectator metrics.
+     *
+     * @param name the qualified name of the table
+     * @param params the get-table parameters
+     */
+    private void logGetTableMetrics(final QualifiedName name,
+                                    final GetTableServiceParameters params) {
+        registry.counter(
+            "metacat.tableService.getTable",
+            "catalog", name.getCatalogName(),
+            "database", name.getDatabaseName(),
```

Comment:
From requestWrapper, the registry metric `metacat.server.counter.request` have tags - catalogName, databaseName, tableName.